src/const/admin/donate.ts

```diff
@@ -0,0 +1,130 @@
+const getMinDigitsError = (count: number) => `Введено менше ${count} цифр`;
+const getMaxDigitsError = (count: number) => `Введено більше ${count} цифр`;
+const getMinSymbolsError = (count: number) => `Введено менше ${count} символів`;
+const getMaxSymbolsError = (count: number) => `Введено більше ${count} символів`;
+const getRequiredError = (field: string) => `${field} є обов'язковим полем`;
+
```

Comment:
probably repeated text can be extracted to constant like `COMMON_TEXT`
```suggestion
 const COMMON_TEXT = {
   ADD_NEW_BTN: 'Додати нові реквізити',
   DELETE_CONFIRM: 'Видалити реквізити?',
   PLACEHOLDER_BANK_NAME: 'Введіть назву банку',
   TITLE_IBAN: 'IBAN',
 };
```

Reply:
Okay. In my opinion, it is not necessary to write all the terms that coincide, for clarity and simplicity of changing a separate string to some option 1 (for example, other support options, and the bank remains as it is). I wrote more general and abstract constants in COMMON_TEXT.